app-tests/nek5000/nek5000_check.py

```diff
@@ -0,0 +1,47 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class nek5000_check(rfm.RegressionTest):
+    valid_systems = ['lumi']
+    valid_prog_environs = ['cpeGNU']
+    modules = ['partition/C', 'Nek5000']
+    executable = './nek5000'
+    strict_check = False
+    descr = (f'Nek500 test')
+    maintainers = ['']
+    build_system = 'CustomBuild'
+    #build_locally = False
+
+    @run_after('init')
+    def setup_run(self):
+        self.num_tasks = 32
```

Comment:
This can be set directly on the class body.

Reply:
Done with a9924c2